test/support/annotations/matched_annotation.rb

```diff
@@ -0,0 +1,68 @@
+module Annotations
+  class MatchedAnnotation
+    attr_accessor :annotation, :requests, :method, :path, :description, :title,
+                  :ignore_duplicates, :starts_with, :ignored_count, :request_body
```

Comment:
Something to think about, if not all of these are being set outside this class they could be reduced to `attr_reader` and further reduced to just using `@` if they aren't being read outside the class. This would reduce the "interface" surface of the class.